pkg/apis/eksctl.io/v1alpha5/defaults.go

```diff
@@ -167,6 +167,7 @@ func setNodeGroupBaseDefaults(ng *NodeGroupBase, meta *ClusterMeta) {
 	if ng.InstanceSelector == nil {
 		ng.InstanceSelector = &InstanceSelector{}
 	}
+	normalizeAMIFamily(ng)
```

Comment:
Can you add a test for this? Also, people add the output of your manual testing in the PR description

Reply:
added unit test for case normalization

Reply:
@Himangini please let me know if we need anything else before we can merge this.

Reply:
Can you update your branch and check `Allow edits and access to secrets by maintainers`?

Reply:
I have updated the branch, but for some reason I don't see the option to allow edits/access to secrets. 
![Screen Shot 2022-10-03 at 1 47 41 PM](https://user-images.githubusercontent.com/613236/193655065-d23cf97e-69d0-4010-82e1-2bef93b49a0c.png)

Reply:
this is perhaps due to your main branch. Please see the guidelines here https://github.com/weaveworks/eksctl/blob/main/CONTRIBUTING.md#pr-submission-guidelines

Reply:
I have attempted creating the PR from a different (not main) branch as well, and it still does not provide the option. Would you like me to close this PR and start a new one from a different branch?

Reply:
I'll make an exception this time, please create a branch for your future contributions 👍🏻